tidb-cloud/set-up-vpc-peering-connections.md

```diff
@@ -127,13 +127,17 @@ Use either of the following two options to approve and configure the VPC peering
 
     ```bash
     # Creates route table rules.
-    aws ec2 describe-route-tables --region "$app_region" --filters Name=vpc-id,Values="$app_vpc_id" --query 'RouteTables[*].RouteTableId' --output text | xargs -n 1 |  while read row
+    aws ec2 describe-route-tables --region "$app_region" --filters Name=vpc-id,Values="$app_vpc_id" --query 'RouteTables[*].RouteTableId' --output text | tr "\t" "\n" | while read row
     do
         app_route_table_id="$row"
         aws ec2 create-route --route-table-id "$app_route_table_id" --destination-cidr-block "$tidbcloud_project_cidr" --vpc-peering-connection-id "$pcx_tidb_to_app_id"
     done
     ```
 
+    > **Note:**
+    >
+    > If the `An error occurred (MissingParameter) when calling the CreateRoute operation: The request must contain the parameter routeTableId` error occurs when you try to create a route table rule, the route table rules might be successfully created. In this case, you can check the rules and ignore the error.
```

Comment:
"might be successfully created"? or "might be not successfully created"

Reply:
might be. Even if the error occurs, the route table rule are still created.

Reply:
got it now. I added a suggested change. PTAL